plugin/src/main/scala/chisel3/internal/plugin/BundleComponent.scala

```diff
@@ -247,7 +246,7 @@ private[plugin] class BundleComponent(val global: Global, arguments: ChiselPlugi
         val isAutoCloneType: Boolean = isAnAutoCloneType(record.symbol)
 
         if (!isAutoCloneType) {
-          warnOnCloneType(record.impl.body)
+          errorOnCloneType(record.impl.body)
           // Other than warning, there is nothing to do on Records that don't mixin AutoCloneType
           return super.transform(record)
         }
```

Comment:
All of this should be deleted because users no longer should be mixing in `AutoCloneType`, it needs to be done automatically even when they don't mix it in.